app/generated_resources.grd

```diff
@@ -7271,16 +7294,25 @@ Keep your key file in a safe place. You will need it to create new versions of y
         Side Panel Selector
       </message>
       <message name="IDS_SIDE_PANEL_COMPANION_TITLE" desc="The name of the companion feature in the side panel combo box.">
-        Brave
+        Search
+      </message>
+      <message name="IDS_SIDE_PANEL_COMPANION_TOOLBAR_TOOLTIP" desc="The tooltip for the companion feature toolbar button.">
+        Brave Search side panel
```

Comment:
Does search in the side panel work?

Reply:
From what I understand, new side panel features like search will require some amount of integration work on our side before they show up in Brave. cc: @simonhong

Reply:
Yes - you are correct. I built/verified we have that turned off and even if you turn on the flag (brave://flags, search for Side Search) it doesn't work 👍